.github/workflows/playwright-testing.yml

```
@@ -0,0 +1,235 @@
+          SERVER_GRPC_PORT: '7077'
+          SERVER_URL: http://localhost:8888
+          SERVER_AUTH_SET_EMAIL_VERIFIED: 't'
+          SERVER_DEFAULT_ENGINE_VERSION: 'V1'
+          SERVER_LOGGER_LEVEL: debug
+          SERVER_LOGGER_FORMAT: console
+
+    steps:
+      - run: echo "Running Playwright PoC tests for draft PR"
+
+      - name: Define node version
+        uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)